src/backend/oracle-data-api/src/main/resources/occm-openapi-spec.yaml

```diff
@@ -541,6 +541,7 @@ components:
         filingDt:
           type: string
           format: date
+          x-field-extra-annotation: "@com.fasterxml.jackson.annotation.JsonFormat(pattern=\"yyyy-MM-dd'Z'\")"
```

Comment:
filingDt is out by one day when I run my persistence test. I had this problem on the disputantBirthDt field too. You may need to do a similar fix.

Reply:
That's weird. When I tested by actually sending a request to create a dispute and return that, dates were matching. However, you're right, it doesn't match in the persistence test. As you mentioned this can be fixed in a different PR following a similar approach to disputantBirthDt field but most likely filingDt field is going to be removed completely 
 anyways since there is no use in the application so we may ignore it for now.